shared/module/MakeShared/src/Logging/OpgJsonFormatter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakeShared\Logging;
+
+use Monolog\Formatter\NormalizerFormatter;
+use Monolog\LogRecord;
+
+class OpgJsonFormatter extends NormalizerFormatter
+{
+    public ?string $requestMethod = null;
+    public ?string $requestPath = null;
+
+    /**
+     * {@inheritDoc}
+     */
+    public function format(LogRecord $record): string
+    {
+        $original = parent::format($record);
+
+        $record = [
+            'time' => $original['datetime'],
```

Comment:
We probably need to check that non of the original format is used in metrics/alerts before merge.